tests/unit/unit_t_db_base.py

```diff
@@ -100,14 +100,18 @@ def setUpClass(cls):
                 return
 
             if os.environ.get('DB_USER') is None:
+                # Get couchdb docker node name
```

Comment:
I think the standard is `nonode@nohost` but I guess this future proofs it a bit :)